test/images/image-util.sh

```diff
@@ -134,7 +135,7 @@ build() {
     if [[ -f ${img_folder}/Makefile ]]; then
       # make bin will take care of all the prerequisites needed
       # for building the docker image
-      make -C "${img_folder}" bin OS="${os_name}" ARCH="${arch}" TARGET="${temp_dir}"
+      make -C "${img_folder}" bin OS="${os_name}" ARCH="${arch}" TARGET_NAME="${temp_dir}"
```

Comment:
without this spellcheck got confused. Perhaps there is a target defined in one of init's dependencies which confused it. So needed to rename this env variable

Reply:
TARGET is used by all the images
```
est/kubemark/pre-existing/README.md:KUBEMARK_IMAGE_MAKE_TARGET="push"
test/images/nonewprivs/Makefile:TARGET ?= $(CURDIR)
test/images/sample-apiserver/Makefile:TARGET ?= $(CURDIR)
test/images/sample-apiserver/Makefile:  docker run --rm -i -v "${TARGET}:${TARGET}:Z" registry.k8s.io/build-image/kube-cross:${KUBE_CROSS_VERSION} \
test/images/sample-apiserver/Makefile:                  find /go/bin -name sample-apiserver* -exec cp {} ${TARGET}/sample-apiserver \;"
test/images/regression-issue-74839/Makefile:TARGET ?= $(CURDIR)
test/images/pets/redis-installer/Makefile:TARGET ?= $(CURDIR)
test/images/pets/zookeeper-installer/Makefile:TARGET ?= $(CURDIR)
test/images/pets/peer-finder/Makefile:TARGET ?= $(CURDIR)
test/images/image-util.sh:      make -C "${img_folder}" bin OS="${os_name}" ARCH="${arch}" TARGET="${temp_dir}"
test/images/image-util.sh:  docker run --rm -v "${TARGET}:${TARGET}:Z" -v "${KUBE_ROOT}":/go/src/k8s.io/kubernetes:Z \
test/images/image-util.sh:                CGO_ENABLED=0 ${arch_prefix} GOOS=${OS} GOARCH=${ARCH} go build -a -installsuffix cgo --ldflags \"-w ${LD_FLAGS:-}\" -o ${TARGET}/${SRC} ./$(dirname "${SRC}")"
test/images/resource-consumer/Makefile:TARGET ?= $(CURDIR)
test/images/agnhost/Makefile:TARGET ?= $(CURDIR)
test/images/busybox/Makefile:TARGET ?= $(CURDIR)
test/images/sample-device-plugin/Makefile:TARGET ?= $(CURDIR)
test/images/apparmor-loader/Makefile:TARGET ?= $(CURDIR)
```

Reply:
fixed the shellcheck warning differently. I am still puzzles where the warning is coming from and how adding the init.sh breaks it.